.github/workflows/npm-audit-fix.yaml

```
@@ -38,21 +45,46 @@ jobs:
+            done
+            cd "$GITHUB_WORKSPACE/ui/dashboard" && npm audit --omit=dev
+            cd "$GITHUB_WORKSPACE/ui/extension" && npm audit --omit=dev
+            echo "force_used=true" >> "$GITHUB_OUTPUT"
+          fi
+
+      - name: Prepare PR body
+        id: pr-body
+        run: |
+          BODY="Automated PR to fix npm audit vulnerabilities.
+
+Generated by the [npm audit fix](https://github.com/argoproj-labs/gitops-promoter/actions/workflows/npm-audit-fix.yaml) workflow."
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)